src/pages/Lyrics.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+
+export default function Lyrics() {
+  const [lyrics, setLyrics] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch("http://localhost:4000/api/lyrics?artist=bts&track=butter")
```

Comment:
This won't work in your deployment. You should check mikes repository for instance. Or ask Martin, he can explain you how you can do it with vercel as well